Model/DbModels/Request.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Model.Enums;
+
+namespace Model.DbModels
+{
+    public class Request : DbModel
+    {
+        [Required]
+        [ForeignKey("User")]
+        public int UserID { get; set; }
+        public User User { get; set; }
+        [ForeignKey("Songs")]
+        public int? SongID { get; set; }
```

Comment:
Waarom deze `int?` ?

Reply:
Omdat er twee types requests zijn, als het een artistrequest is hoeft er geen song meegegeven te worden.

Reply:
Maar dat gaat goed met een foreignkey?
Als iets null is?

Reply:
We hadden de nullcheck voordat @sanderpol de foreignkey had toegevoegd vgm. Bij het aanmaken van een request ging het zeuren dat het een SongID nodig had en die mocht niet null zijn.

Reply:
Ja precies, nu zeg je dat SongID nullable kan zijn. Maar je mag toch niet nullable en foreignkey hebben?

Reply:
Ja! [Dat kan](https://stackoverflow.com/questions/7573590/can-a-foreign-key-be-null-and-or-duplicate)

Reply:
:)

Reply:
![tick](https://user-images.githubusercontent.com/9300386/101176696-20744200-3647-11eb-8a96-e0820c41a294.png)